src/main/kotlin/com/lovelysystems/gradle/LSGit.kt

```diff
@@ -96,10 +96,15 @@ class LSGit(val dir: File) {
         }
     }
 
-    fun latestLocalGitTagVersion(): Version? {
-        return gitCmd("tag", "-l", "--sort=-v:refname").lines().find { isProductionVersion(it) }?.let {
-            Version.fromIdent(it)
-        }
+    fun latestLocalGitTagVersion(releaseVersion: Version? = null): Version? {
```

Comment:
pls change the function name to reflect that this returns not the latest version but only the latest patch version